lib/client/auth/brokerServerConnection.ts

```diff
@@ -0,0 +1,51 @@
+import { PostFilterPreparedRequest } from '../../common/relay/prepareRequest';
+import { CONFIGURATION } from '../../common/types/options';
+import version from '../../common/utils/version';
+import {
+  HttpResponse,
+  makeRequestToDownstream,
+} from '../../hybrid-sdk/http/request';
+import { Role } from '../types/client';
+
+export interface BrokerServerConnectionParams {
+  connectionIdentifier: string;
+  brokerClientId: string;
+  authorization: string;
+  role: Role;
+  serverId: number;
+}
+export const renewBrokerServerConnection = async (
+  brokerServerConnectionParams: BrokerServerConnectionParams,
+  clientConfig: CONFIGURATION,
+): Promise<HttpResponse> => {
+  const apiHostname = clientConfig.API_BASE_URL;
```

Comment:
Does this replace or otherwise override `BROKER_DISPATCHER_BASE_URL`?

Reply:
it replaces it entirely. It's only here as backward compatibility in case some folks have it configured that way.